contracts/VeSugar.vy

```diff
@@ -146,6 +155,11 @@ def _byId(_id: uint256) -> VeNFT:
   amount, expires_at, perma = self.ve.locked(_id)
   last_voted: uint256 = 0
 
+  timepoint: uint256 = convert(self.gov.clock(), uint256)
+  governance_amount: uint256 = self.gov.getVotes(_id, timepoint - 1)
```

Comment:
Was taking a closer look at this, do we actually need to call `clock()`, a simple `block.timestamp` should be enough, no?